src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -240,6 +243,7 @@ async def create_privacy_request(
                         "message": "",
                     },
                 )
+                # do we want approval email sent for automatic approvals?
                 queue_privacy_request(privacy_request.id)
```

Comment:
good question, my opinion would be "no", at any rate, let's remove this code comment and just run by @seanpreston

Reply:
Definitely not! Thanks for clarifying